server/service/transport_error.go

```diff
@@ -61,7 +63,14 @@ type existsErrorInterface interface {
 	IsExists() bool
 }
 
-// encode error and status header to the client
+// errorEncoder encodes the error on the response.
+func errorEncoder(logger kitlog.Logger) kithttp.ErrorEncoder {
+	return func(ctx context.Context, err error, w http.ResponseWriter) {
+		level.Error(logger).Log("err", err)
```

Comment:
This one logs other errors like authentication errors (that are not `(x ...) error() err` errors).